microsoftgraph/microsoft-graph-explorer-v4
# Fix: Speed up autocomplete

## Overview
Fixes #2005 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
A debounce function has been added that allows you to type, and fetches suggestions 200 milliseconds after you've stopped typing

## Testing Instructions

* How to test this PR
Check out this branch
Type queries on the query textbox
Notice that you can type and delete queries without the UI hanging
